api/v1alpha1/watermarkpodautoscaler_types.go

```diff
@@ -64,11 +64,17 @@ type WatermarkPodAutoscalerSpec struct {
 	// ScaleUpLimitFactor == 0 means that upscaling will not be allowed for the target.
 	ScaleUpLimitFactor *resource.Quantity `json:"scaleUpLimitFactor,omitempty"`
 
+	// +kubebuilder:validation:Minimum=0
+	UpscaleEvaluateAboveWatermarkSeconds int32 `json:"upscaleEvaluateAboveWatermarkSeconds,omitempty"`
```

Comment:
can it be `uint32` ?

Reply:
sure, we have not been enforcing this in the CRD but in the controller thus far. I can update.

Reply:
will actually leave as is, I don't like the behavior this introduces

```	
canScaleDelay := int32(decisionDelay) - int32(now.Sub(wasOutOfBounds.LastTransitionTime.Time).Seconds()) // what we had before
canScaleDelay2 := decisionDelay - uint32(now.Sub(wasOutOfBounds.LastTransitionTime.Time).Seconds()) // what we have with uint
[...]
{"level":"info","ts":1652200295.957087,"logger":"All scale authorized","msg":"TEST","canScaleDelay":-30,"canScaleDelay2":4294967266}
```